### DISCUSSION TITLE
Automatic Updates

### DISCUSSION THREAD

**Discussion by trymeouteh** [ScoopInstaller/Scoop#5467] [Help and Questions]
Does scoop updates application automatically in the background?

**Comment by HUMORCE** [comment]
The official bucket checks and updates manifests via gh action.

refer:
https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate
https://github.com/ScoopInstaller/GithubActions
https://github.com/ScoopInstaller/BucketTemplate
